apis/kueue/v1beta1/workload_types.go

```diff
@@ -639,6 +643,10 @@ const (
 	// due to non-recoverable node failures.
 	WorkloadEvictedDueToNodeFailures = "NodeFailures"
 
+	// WorkloadRemovedViaWorkloadSliceAggregation indicates that the workload instance was
+	// removed (typically from the internal cache only) as a result of workload slice aggregation.
+	WorkloadRemovedViaWorkloadSliceAggregation = "RemovedViaWorkloadSliceAggregation"
```

Comment:
```suggestion
	// WorkloadSliceReplacement indicates that the workload instance was
	// replaced with a new slice.
	WorkloadSliceReplacement = "WorkloadSliceReplacement"
```
To align with KEP, and I think we shouldn't say Removed if it still exists.

Reply:
Update per suggestion, PTAL, @mimowo

Reply:
Yes